app/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "__MSG_appName__",
   "short_name": "__MSG_appName__",
-  "version": "7.2.3",
+  "version": "7.2.4",
```

Comment:
This change isn't needed - we bump the version when finalizing a release, rather than doing it in the first PR after we release.